postcss-selector-parser.d.ts

```diff
@@ -211,8 +211,8 @@ declare namespace parser {
         Child extends Node = Node
     > extends Base<Value> {
         nodes: Array<Child>;
-        append(selector: Selector): this;
-        prepend(selector: Selector): this;
+        append(selector: Child): this;
+        prepend(selector: Child): this;
```

Comment:
I think this was a typo.

`append`/`prepend` have similar implementations as `insertAfter`/`insertBefore` and those have type `Child`